lib/parsers/log-parser.js

```diff
@@ -18,6 +18,12 @@ const ERROR_PATTERN = new RegExp('' +
   '(.+?)\\.?$'           // Message text, the ending period is optional for MiKTeX
 )
 
+// Pattern for obtaining the last macro in a line
+// Meant for catching the undefined control sequence
+const LAST_MACRO_PATTERN = new RegExp('' +
+  '(\\\\[a-zA-Z@]+)$' // Macro name
```

Comment:
I'm not confident as to what ideally should constitute a legal macro name in
this setting.

One of the biggest gotchas here is that in LaTeX, even space can be made
a letter-character (thus allowed in macro names). However, that would be
weirdly inconvenient. The other is that any character can be made catcode 0,
that is, the escape character (by default `\`).

I would argue that it's be better to just look at the last space and
go from there, as it is rather unlikely that one would redefine the space
as such:

```suggestion
  '([^ ]+)$' // Macro name
```

An issue with the definition here is that I din't include underscores `_`, 
which will be very widely used in macro names come LaTeX3.

```suggestion
  '(\\\\[a-zA-Z@_]+)$' // Macro name
```

Reply:
Actually, I think the best way would be to assume that `\` is the escapechar and

```suggestion
  '(\\\\[^\\\\]+)$' // Macro name
```